templates/repo/diff.tmpl

```diff
@@ -105,14 +105,14 @@
                 {{else}}
                 <table>
                     <tbody>
-                        {{range $j, $section := $file.Sections}}
-                        {{range $k, $line := $section.Lines}}                        
-                        <tr class="{{DiffLineTypeToStr .Type}}-code nl-{{$i}} ol-{{$i}}">
+                        {{range .Sections}}
+                        {{range $k, $line := .Lines}}
+                        <tr class="{{DiffLineTypeToStr .Type}}-code nl-{{$k}} ol-{{$k}}">
                             <td class="lines-num lines-num-old">
-                                <span rel="diff-{{Add $i 1}}L{{$j}}{{$k}}">{{if $line.LeftIdx}}{{$line.LeftIdx}}{{end}}</span>
+                                <span rel="{{if $line.LeftIdx}}diff-L{{Sha1 $file.Name}}{{$line.LeftIdx}}{{end}}">{{if $line.LeftIdx}}{{$line.LeftIdx}}{{end}}</span>
```

Comment:
I think this L is on the wrong place, shouldn't it be after the filename just like the R?